fe/fe-core/src/test/java/com/starrocks/sql/ast/GrantRevokeImpersonateStmtTest.java

```diff
@@ -70,12 +74,23 @@ public void testNormal() throws Exception {
         com.starrocks.sql.analyzer.Analyzer.analyze(stmt, ctx);
         Assert.assertEquals("GRANT IMPERSONATE ON 'user2'@'%' TO 'user1'@'%'", stmt.toString());
 
+        stmt = (GrantImpersonateStmt) com.starrocks.sql.parser.SqlParser.parse(
+                "grant IMPERSONATE on user2 to ROLE role1", 1).get(0);
+        com.starrocks.sql.analyzer.Analyzer.analyze(stmt, ctx);
+        Assert.assertEquals("GRANT IMPERSONATE ON 'default_cluster:user2'@'%' TO ROLE 'default_cluster:role1'", stmt.toString());
```

Comment:
You should rebase the latest main and remove the default_cluster prefix. Or this ut will fail when merge into main branch.

Reply:
rebased & fixed.